zh/get-started.md

```diff
@@ -539,10 +539,10 @@ mysql> select * from information_schema.tikv_region_status where db_name=databas
 ```sql
 mysql> select tidb_version()\G
 *************************** 1. row ***************************
-         tidb_version(): Release Version: v7.1.1
+         tidb_version(): Release Version: v7.5.0
                 Edition: Community
         Git Commit Hash: cf441574864be63938524e7dfcf7cc659edc3dd8
-             Git Branch: heads/refs/tags/v7.1.1
+             Git Branch: heads/refs/tags/v7.5.0
```

Comment:
it's better to update the above `Git Commit Hash`, `UTC Build Time`, etc to the real values after TiDB v7.5.0 is released.